bndtools.core/bndtools.shared.bndrun

```diff
@@ -63,6 +104,7 @@ crossplatform.bundles: \
 	org.osgi.impl.bundle.repoindex.lib,\
 	${help.bundles},\
 	${p2.bundles},\
+	${m2e.bundles},\
```

Comment:
Should we not have a separate launch for m2e testing? After all not everyone will have m2e installed in their eclipse.